requirements.txt

```diff
@@ -1,37 +1,33 @@
-# counterfit
-cmd2==2.2.0
+onnxruntime==1.10.0
+gym==0.19.0                                 # Required for cartpole target
+pyglet                                      # Required for cartpole target
+cmd2==2.2.0                                 # Counterfit terminal
 questionary==1.9.0
-python-Levenshtein==0.12.2
+python-Levenshtein>=0.20.0
+
 rich==12.0.1
 orjson==3.6.4
 flask==2.0.0
-
-# docs
-sphinx==5.1.1
-sphinx-rtd-theme==1.0.0
-
-# frameworks
-adversarial-robustness-toolbox==1.8
-textattack==0.3.4
+optuna
+numpy>=1.18.5
+requests==2.24.0
+scipy==1.9.2
+h5py==3.1.0
+lightgbm==3.3.1
+pillow
+pyyaml
```

Comment:
Versions are missing for some of the packages in `requirements.txt`. Could you update with right versions, if we don't provide strict versioning, it may cause package version dependency issue when a new version published into PyPi

Reply:
Solved.